.drone.yml

```diff
@@ -114,6 +125,17 @@ steps:
     event:
     - tag
 
+- name: docker-build
+  image: plugins/docker
+  settings:
+    dockerfile: package/Dockerfile
+    repo: "rancher/eks-operator"
+    tag: "${DRONE_COMMIT}-arm64"
+    dry_run: true
+  when:
+    event:
+    - pull_request
+
```

Comment:
whats going on in these two ? We removed image build from CI so it gets run on PR now ? Whats reasoning for that. And whats difference between this block and block above.

Reply:
There was a lot of discussion on the installation of Docker in the Dapper image because we were using an old version Docker. See https://github.com/rancher/rancher/blob/master/Dockerfile.dapper#L35-L38 for what it looked like.

The outcome of that was instead of building the image in `scripts/package`, let the plugin handle it. So I added a Docker plugin step that builds the image on pull requests, but doesn't push it (no username nor password is added and `push` is set to `false`).